node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -56,7 +56,8 @@ type WaitForMpathResult struct {
 }
 
 var (
-	TimeOutMultipathFlashCmd = 4 * 1000
+	TimeOutForCmd = 4 * 1000
```

Comment:
consider renaming to something like `TimeOutMultipathCmd`

Reply:
done